assets/js/romo/keypress.js

```diff
@@ -0,0 +1,35 @@
+$.fn.romoKeypress = function() {
+  return $.map(this, function(element) {
+    return new RomoKeypress(element);
+  });
+}
+
+var RomoKeypress = function(element) {
+  this.elem = $(element);
+  this.defaultTriggerOn = 'keydown';
```

Comment:
@kellyredding - We discussed maybe making this `keypress`. I can't remember all the details for each of the events but it might make more sense since the component is called keypress as well.